readthedocs/rtd_tests/tests/test_project_querysets.py

```diff
@@ -79,18 +79,6 @@ def test_subproject_queryset_attributes(self):
         self.assertEqual(ChildRelatedProjectQuerySet.project_field, 'child')
         self.assertTrue(ChildRelatedProjectQuerySet.use_for_related_fields)
 
-    def test_subproject_queryset_as_manager_gets_correct_class(self):
```

Comment:
Is it safe to delete this? We don't want to check that the subproject still gets the right class, just without the `Base`?

Reply:
Test is back!